utilities/CovidDownloader.py

```diff
@@ -0,0 +1,42 @@
+import pandas as pd
+from urllib.request import urlopen
+import json
+import csv
+
+
+class csvDownloader():
+	def __init__(self):
+		pass	
+
+	def find_data(self, countryName):
+		
+
+		df = pd.read_csv(url)
+		global myDict 
```

Comment:
What is the purpose of defining global myDict? If you erase it myDict should still work just the same

Reply:
same comment for all the other globals